src/main/java/com/starkbank/PaymentRequest.java

```diff
@@ -0,0 +1,314 @@
+     * @param due [string]: Payment target date in ISO format.
+     * @param tags [list of strings]: list of strings for tagging
+     * @param amount [long]: PaymentRequest amount. ex: 100000 = R$1.000,00
+     * @param status [string]: current PaymentRequest status.ex: "pending" or "approved"
+     * @param actions [List of <Map<string, string>]: list of actions that are affecting this PaymentRequest.ex: [{"type": "member", "id": "56565656565656, "action": "requested"}]
+     * @param updated [string]: latest update datetime for the PaymentRequest. ex: 2020-12-31
+     * @param created [string]: creation datetime for the PaymentRequest. ex: 2020-12-31
+     */
+    public PaymentRequest(String id, String centerId, Resource payment, String type, String due, String[] tags, Long amount,
+                          String status, List<Map<String, String>> actions, String updated, String created) throws Exception{
+        super(id);
+        this.centerId = centerId;
+        this.payment = payment;
+        this.due = due;
+        this.tags = tags;
+        this.amount = amount;
+        this.status = status;
+        this.actions = actions;
+        this.updated = updated;
+        this.created = created;
+
+        this.type = type;
+        if(type == null) {
+            this.type = PaymentRequest.getType(payment);
+        }
+    }
+
+    /**
+     * PaymentRequest object
+     * <p>
+     * When you initialize a PaymentRequest, the entity will not be automatically
+     * sent to the Stark Bank API. The "create" function sends the objects
+     * to the Stark Bank API and returns the list of created objects.
+     * All parameters are passed in a Map of String and Object object.
+     * <p>
+     * @param data map of parameters for the creation of the PaymentRequest
+     * Parameters:
+     * centerId [string]: target cost center ID. ex: "5656565656565656"
+     * payment [Transfer, BoletoPayment, UtilityPayment, Transaction or dictionary]: payment entity that should be approved and executed.
+     * <p>
+     * Parameters (conditionally required):
+     * type [string]: payment type, inferred from the payment parameter if it is not a map. ex: "transfer", "boleto-payment"
+     * <p>
+     * Parameters (optional):
+     * due [string]: Payment target date in ISO format. ex: 2020-12-31
+     * tags [list of strings]: list of strings for tagging
+     */
+    @SuppressWarnings("unchecked")
+    public PaymentRequest(Map<String, Object> data) throws Exception{
+        super(null);
+
+        HashMap<String, Object> dataCopy = new HashMap<>(data);
+        this.centerId = (String) dataCopy.remove("centerId");
+        this.payment = (Resource) dataCopy.remove("payment");
+        this.due = (String) dataCopy.remove("due");
+        this.tags = (String[]) dataCopy.remove("tags");
+        this.amount = (Long)((Number) dataCopy.remove("amount"));
```

Comment:
Só adicionei um cast pra long aqui, porque tava dando pau com o método .LongValue quando amount não era passado.

Reply:
Dava isso aqui:
 Exception in thread "main" java.lang.NullPointerException
	at gfg.main(gfg.java:11)